src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts

```diff
@@ -545,5 +593,21 @@ export class WorkspaceSavedObjectsClientWrapper {
     };
   };
 
-  constructor(private readonly permissionControl: SavedObjectsPermissionControlContract) {}
+  constructor(
+    private readonly permissionControl: SavedObjectsPermissionControlContract,
+    private readonly options: {
+      config$: Observable<WorkspacePluginConfigType>;
+    },
+    private readonly applicationConfig: boolean
+  ) {
+    this.options?.config$.subscribe((config) => {
+      this.config = config;
+    });
+    this.options?.config$
+      .pipe(first())
+      .toPromise()
+      .then((config) => {
+        this.config = config;
+      });
```

Comment:
Not needed I suppose. Should move to postAuth.

Reply:
The useless code has been deleted.